File: dclib/core/src/main/java/dclib/epf/parts/TransformPart.java
package dclib.epf.parts;

import dclib.physics.Transform;

public final class TransformPart {

	private final Transform transform;

	public TransformPart(final Transform transform) {
		this.transform = transform;
	}

	public final Transform getTransform() {
		return transform;
	}

}

File: dclib/core/src/main/java/dclib/epf/parts/TransformPart.kt
package dclib.epf.parts

import dclib.eventing.Property
import dclib.physics.Transform

class TransformPart(transform: Transform) {
    val transformChanged get() = transformProperty.changed
    private val transformProperty = Property(transform)
    var transform
        get() = transformProperty.value
        set(value) { transformProperty.value = value }
}